web-app/js/portal/details/SubsetPanel.js

```diff
@@ -18,7 +18,6 @@ Portal.details.SubsetPanel = Ext.extend(Ext.Container, {
 
         var config = Ext.apply({
             title: OpenLayers.i18n('subsetPanelTitle'),
-            hideMode: 'offsets', // fixes #1278
```

Comment:
Yes this bug wont apply do to this massive layout change. New bugs will apply :)